src/main/scala/org/broadinstitute/dsde/workbench/avram/config/DbcpDataSourceConfig.scala

```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.dsde.workbench.avram.config
+
+case class DbcpDataSourceConfig(driverClassName: String,
```

Comment:
Nice. I like making this more explicit. Can we have this case class and its `ValueReader` in the same file?

Reply:
We could - I was following the Leo and Sam format

Reply:
They're already tightly coupled because they enumerate the same fields so I think they belong as close together as possible.